aries-site/src/data/structures/components.js

```diff
@@ -821,6 +821,28 @@ export const components = [
     relatedContent: ['Page Layouts', 'Content Layouts', 'Box', 'Main'],
     tags: ['page', 'page content'],
   },
+  {
+    name: 'PageHeader',
+    available: true,
+    category: 'Layouts',
+    description: `A required page component that provides users context of 
+    their current location and a consistent experience across all HPE applications.`,
+    preview: {
+      image: {
+        src: {
+          light: '/componentImages/component-preview-page.svg',
+          dark: '/componentImages/component-preview-page-invert.svg',
+        },
+        alt: 'HPE Page Preview',
+        fit: 'contain',
+      },
+    },
+    seoDescription: `A required page component that provides users context of 
+    their current location and a consistent experience across all HPE applications.`,
+    sections: ['Guidance', 'Page Content'],
```

Comment:
Add:
Anatomy
Responsive Behavior and Content Prioritization
Title
Subtitle
Reverse Anchor
Page-Level Actions
Responsive Behavior of Actions

Reply:
Done.